### ISSUE TITLE
Add the material-icons class to icon-button

### ISSUE THREAD

**Issue by ticamacho** [matsp/material-components-vue#133]
As a proposed improvement to the icon-button component, should we add the "material-icons" class to the button element?

I know it's not compulsory according to the framework, but without this class the icons won't rendered correctly (using the toggle-on-content and toggle-off-content props).

This means that we always need to pass the class value from the parent that is using the component.

**Comment by matsp** [comment]
@ticamacho 

I think the google team made this desicion to allow other icos like 'Font Awesome'.
I implemented the toggling behavior also with slots in favor of decoupling the components. 

So its possible to use SVGs, <m-icon/> or other elements for toggling.

Whats your opinion?

**Comment by ticamacho** [comment]
Yes that makes sense. My expectation there however was that it would be ready for use with material icons, so I was a bit confused in the beginning. Perhaps, just add a mention into the documentation? Thanks.
